openhtf/io/frontend/src/app/station.service.ts

```diff
@@ -97,12 +97,12 @@ export class StationService extends SubscriptionService {
    */
   onmessage(msg) {
     this.reachable = true;
-    if (!msg) {
+    let parsedData = JSON.parse(msg.data);
```

Comment:
Possibly. Frontend client needs better console debug logging overall. Filed #388 to track general console logging improvements in the web client.